src/server/pfs/server/driver_v2.go

```diff
@@ -524,3 +520,310 @@ func (d *driverV2) walkFile(pachClient *client.APIClient, file *pfs.File, cb fun
 		return cb(fi)
 	})
 }
+
+func (d *driverV2) deleteRepo(txnCtx *txnenv.TransactionContext, repo *pfs.Repo, force bool) error {
+	ctx := txnCtx.ClientContext
+	if err := d.storage.WalkFileSet(ctx, repo.Name, func(p string) error {
+		return d.storage.Delete(ctx, p)
+	}); err != nil {
+		return err
+	}
+	return d.driver.deleteRepo(txnCtx, repo, force)
+}
+
+func (d *driverV2) deleteAll(txnCtx *txnenv.TransactionContext) error {
```

Comment:
What is going on with the auth stuff in this? Is `includeAuth` a global variable, and why do we filter out auth errors on the `deleteRepo` call?

Reply:
I'm not sure this is all just copied from the existing driver, with calls to the new storage layer added in.